modules/mobile/docs/openapi.yaml

```diff
@@ -2435,6 +2435,78 @@ paths:
       security:
         - Bearer: []
       summary: /v0/messaging/health/folders/{id}/messages
+  /v0/messaging/health/folders/{id}/threads:
+    get:
+      description: List of threads in a secure messaging folder
+      operationId: get_folder_threads
+      parameters:
+        - description: The id of the folder that threads are being retrieved from
+          in: path
+          name: folderId
+          required: true
+          schema:
+            type: string
+        - description: The size of the pagination you want
+          in: query
+          name: pageSize,
+          required: true
+          schema:
+            type: string
+        - description: The page number to get based on your page size
+          in: query
+          name: page,
+          required: true
+          schema:
+            type: string
+        - description: The field to sort the results by
+          in: query
+          name: sortField,
+          required: true
+          schema:
+            type: string
+            enum: [ SENDER_NAME, RECIPIENT_NAME, SENT_DATE, DRAFT_DATE ]
+        - description: The order to sort the results by
+          in: query
+          name: sortOrder,
+          required: true
+          schema:
+            type: string
+            enum: [ ASC, DESC ]
+        - $ref: "#/components/parameters/InflectionHeader"
+      responses:
```

Comment:
The question has been coming up recently of "should these docs define all possible responses?" That would mean that every single entry in this file should have a section for 500, 502, 503, etc. I don't think you need to make any changes, but we do need to have that conversation sometime and this PR is reminding me. I'm going to file a ticket today for figuring that out.

Reply:
Ya I was going to wait until that discussion comes to a conclusion

Reply:
I've created a ticket for it, so I'd say just skip it for now.